[BUG] Plugin Build Fails Due to missing `nebula` Dependencies.

**Describe the bug**
https://github.com/opensearch-project/security/actions/runs/3415191485/jobs/5684117326
```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'opensearch-security'.
> Could not resolve all files for configuration ':classpath'.
   > Could not resolve com.netflix.nebula:nebula-core:3.0.0.
     Required by:
         project : > org.opensearch.gradle:build-tools:3.0.0-SNAPSHOT:20221103.083150-511 > com.netflix.nebula:nebula-publishing-plugin:4.4.4
      > Could not resolve com.netflix.nebula:nebula-core:3.0.0.
         > Could not get resource 'https://d1nvenhzbhpy0q.cloudfront.net/snapshots/lucene/com/netflix/nebula/nebula-core/3.0.0/nebula-core-3.0.0.pom'.
            > Could not GET 'https://d1nvenhzbhpy0q.cloudfront.net/snapshots/lucene/com/netflix/nebula/nebula-core/3.0.0/nebula-core-3.0.0.pom'. Received status code 403 from server: Forbidden
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -99,6 +99,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - OpenJDK Update (October 2022 Patch releases) ([#4997](https://github.com/opensearch-project/OpenSearch/pull/4997))
 - Upgrade zookeeper dependency in hdfs-fixture ([#5007](https://github.com/opensearch-project/OpenSearch/pull/5007))
 - Update Jackson to 2.14.0 ([#5105](https://github.com/opensearch-project/OpenSearch/pull/5105))
+- Bump nebula-publishing-plugin from 4.4.4 to 4.6.0 ([#5127](https://github.com/opensearch-project/OpenSearch/pull/5127))
 
 ### Changed
 - Dependency updates (httpcore, mockito, slf4j, httpasyncclient, commons-codec) ([#4308](https://github.com/opensearch-project/OpenSearch/pull/4308))

--- a/buildSrc/build.gradle
+++ b/buildSrc/build.gradle
@@ -106,7 +106,7 @@ dependencies {
   api 'org.apache.commons:commons-compress:1.21'
   api 'org.apache.ant:ant:1.10.12'
   api 'com.netflix.nebula:gradle-extra-configurations-plugin:7.0.0'
-  api 'com.netflix.nebula:nebula-publishing-plugin:4.4.4'
+  api 'com.netflix.nebula:nebula-publishing-plugin:4.6.0'
   api 'com.netflix.nebula:gradle-info-plugin:11.3.3'
   api 'org.apache.rat:apache-rat:0.13'
   api 'commons-io:commons-io:2.7'

